_episodes/12-skim-milk.md

```diff
@@ -10,12 +10,11 @@ hidden: false
 keypoints:
   - Making jobs aware of each other is pretty easy.
```

Comment:
How about "We can specify dependencies with the `needs` value"?

The keypoints should create a cheatsheet, so I'd rather have it point out few specific things than being general

Reply:
Perhaps something similar can be said for artifacts? Or just "Artifacts are files created by the CI that are offered for download and inspection" or something like that

Reply:
How about this > 
 We can specify dependencies between jobs running in a series using the `needs` value. 

since we would need to mention about jobs running in a sequence